src/redux/messageSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const api = 'http://localhost:3000/api/v1/messages';
```

Comment:
- [ ] Good project implementation :clap:  Unfortunately The React app can not use the url to your backend api to display  a random greeting as expected for this project because both back end and frontend app can not communicate because they are blocked by **CORS policy** as you can see on the screenshot below :arrow_down: Please fix it and make sure that your app display the random messages as expected :+1: 
![Screenshot_20230221_004448](https://user-images.githubusercontent.com/6615394/220221513-f3fdfc33-ecff-4be2-8a9d-102f41f93b6d.png)